tensorflow_serving/example/inception_inference.cc

```diff
@@ -208,10 +212,9 @@ InceptionServiceImpl::InceptionServiceImpl(
   // the numbers are extremely performance critical and should be tuned based
   // specific graph structure and usage.
   tensorflow::serving::StreamingBatchScheduler<Task>::Options scheduler_options;
-  scheduler_options.thread_pool_name = "inception_service_batch_threads";
-  // TODO(27776734): Current exported model supports only batch_size=1
-  // See inception_export.py for details.
-  scheduler_options.max_batch_size = 1;
+  scheduler_options.batch_timeout_micros = 1000 * 1000;  // 1 second
+  scheduler_options.num_batch_threads = 4;
+  scheduler_options.max_batch_size = 22;
```

Comment:
Maybe add a comment on how you selected 22 as the max batch size?

Similar for num_batch_threads: are you aiming to match the server machine's number of GPU devices or ??